### ISSUE TITLE
Config files not correctly synced

### ISSUE THREAD

**Issue by paneil** [Skidamek/AutoModpack#136]
### Bug description

When changing the contents of a config file in the modpack and updating it, it does not update the config file for the client. Not quite sure if this is a bug or a missing feature but since `"editable"` is set to `false` for the config file, I'd say AutoModpack should update the client's config as the content is different from the modpacks.

`[main/INFO]: Skipping* already downloaded file: /config/paxi/resourcepack_load_order.json`

Modpack resourcepack_load_order.json content:
```
{
  "loadOrder": ["Xaero's Minimap - Mod Icons.zip", "EmbellishedStone-1.19.3-1.0.0.zip", "MandalasGUI_Legacy+Modded_Dakmode_v3.8.8.zip", "NewDefault+v1.66[MC1.19-1.19.2].zip", "Incendium_Optional_Resourcepack_1.19.4_v5.3.5.zip"]
}
```

Client resourcepack_load_order.json content:
```
{
  "loadOrder": ["Xaero's Minimap - Mod Icons.zip", "EmbellishedStone-1.19.3-1.0.0.zip", "NewDefault+v1.66[MC1.19-1.19.2].zip", "Incendium_Optional_Resourcepack_1.19.4_v5.3.5.zip", "MandalasGUI_Legacy+Modded_Dakmode_v3.8.8.zip"]
}
```

### Steps to reproduce

1. Make sure the modpack is updated with the config already downloaded client-side.
2. Change the contents of it server-side
3. Update modpack
4. Observe how AutoModpack skips the updating/downloading the config file

### Expected behavior

The config file updates

### Actual behavior

It is skipped

### Minecraft version & Mod Loader

1.19.2 fabric

### Operating system

Windows 11

### AutoModpack version

3.3.6

### Check list

- [X] I have verified that the issue persists in the latest version of the mod.
- [X] I have searched the existing issues and confirmed that this is not a duplicate.

**Comment by Skidamek** [comment]
Yoooo figured that out lol this is soo dumb

**Comment by Skidamek** [comment]
fixed in 3.3.7
